src/lando/api/legacy/workers/landing_worker.py

```diff
@@ -215,10 +229,31 @@ def apply_patch(revision: Revision):
 
         self.update_repo(repo, job, scm, job.target_commit_hash)
 
+        if job.is_pull_request_job and job.handover_repo and job.handover_repo.is_try:
+            self.add_try_task_config(scm)
+            self.convert_patches_to_diff(scm, job)
+            job.target_repo = job.handover_repo
+            job.handover_repo = None
+            job.save()
+            job.revisions.update(pull_number=None)
```

Comment:
Add comment describing what is happening here: after the patches are generated, the job is converted from a pull-request "git" landing job to a legacy hg landing job targeting the try repo.